selene-lib/src/rules/limit_function_complexity.rs

```diff
@@ -0,0 +1,272 @@
+use super::*;
+use crate::ast_util::range;
+use std::convert::Infallible;
+
+
+use full_moon::{
+    ast::{self, Ast},
+    // node::Node,
+    visitors::Visitor,
+};
+
+use serde::Deserialize;
+
+#[derive(Clone, Copy, Deserialize)]
+pub struct LimitFunctionComplexityConfig {
+    maximum_complexity: u16,

```

Comment:
Nit: any reason for this to be only a u16?

Reply:
Not really,other than I've never seen a single function exceed ~800 cyclomatic complexity in ~25 years of measuring complexity on 10+ languages.